raiden/network/proxies/service_registry.py

```diff
@@ -36,11 +36,11 @@ def __init__(
 
         self.contract_manager = contract_manager
         check_address_has_code(
-            jsonrpc_client,
-            service_registry_address,
-            CONTRACT_SERVICE_REGISTRY,
-            expected_code=to_bytes(
-                hexstr=contract_manager.get_runtime_hexcode(CONTRACT_SERVICE_REGISTRY)
+            client=jsonrpc_client,
+            address=service_registry_address,
+            contract_name=CONTRACT_SERVICE_REGISTRY,
+            expected_code=decode_hex(
+                contract_manager.get_runtime_hexcode(CONTRACT_SERVICE_REGISTRY)
```

Comment:
why not replace `hexstr=` with `value=` here and in the other places?

Reply:
I see no `value` parameter in the docs: https://eth-utils.readthedocs.io/en/latest/utilities.html#to-bytes-bytes-int-bool-text-str-hexstr-str-bytes

And generally we seem to use `decode_hex` more often.

Reply:
The docs you are linking to are from `to_bytes`. The argument I am referring to is the argument name of `decode_hex()`.

Reply:
Ah, so you mean I should add the `value` keyworkd? because `hexstr` doesn't exist for `decode_hex`

Reply:
yep

Reply:
I'd say it's unnecessary, as it's the only possible parameter for `decode_hex`

Reply:
at the moment. I mentioned it since the code had a keyword parameter before. I leave it up to you. This is definitely a nitpick.

Reply:
Let's keep it woithout the keyword then. I don't think it currently make stuff clearer.